src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -434,10 +471,13 @@ export function getSize(extent: [number, number, number, number]): Size {
 }
 
 export function validateTiffAlignment(tiff: TiffLocation, allowedError = 0.015): boolean {
-  const mapTileIndex = MapSheet.getMapTileIndex(tiff.tileName);
+  const tileName = tiff.tileNames[0];
+  // FIXME
```

Comment:
@blacha what was the FIXME relating to? Does this code need further updates?

Reply:
I don't think we can assume that there is only one tileName since there is a array passed in.

Reply:
fixed in current code